src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,13 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
+
     public User findByEmail(String email) {
+        for (User user : UserService.users) {
```

Comment:
No need to specify the **UserService** here, you should remove it for better readability